test/appstudio_strategy.spec.ts

```diff
@@ -1,11 +1,14 @@
 import { expect } from "chai";
```

Comment:
rename the file to strategy.spec.ts as it is testing several files inside strategy folder

Reply:
done